data-syncs/supported-data-sync-sources/rest-api.md

```diff
@@ -81,6 +93,34 @@ You can learn more about these sections in [Appendix A - Other Sections.](rest-a
 
 <figure><img src="../../.gitbook/assets/image (298).png" alt=""><figcaption><p>Image 2: The Source Tab</p></figcaption></figure>
 
+## Best practices
+
+For JSON objects, use `$.data` in the **Records Root JSONPAth** to access object values. Otherwise, you must append each column in the **Schema** section with `$.data`. 
+
+For example, here is a sample JSON return object:
+```json
+{
+  "data": {
+    "name": "Strawberry",
+    "id": 3,
+    "family": "Rosaceae",
+    "order": "Rosales",
+    "genus": "Fragaria",
+    "nutritions": {
+      "calories": 29,
+      "fat": 0.4,
+      "sugar": 5.4,
+      "carbohydrates": 5.5,
+      "protein": 0.8
+    }
+  }
+}
+```
+To define each column in the source data, you can:
+
+1. Use `$.data` as the **Records Root JSON Path** and  map each column to the attribute name. For example, `$.name`, `$.id`, `$.family`
+1. Use `$` as **Records Root JSON Path** map each column to the attribute name to `$.data.{attribute}`. For example, `$.data.name`, `$.data.id`, `$.data.family`
```

Comment:
Might be nice to include a path to iterate example separately here.

Reply:
I spent a few hours trying various examples that can feed into this and should help clarify all behaviour